media-sound/libopenmpt/libopenmpt-0.2.7386~beta20.3.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="OpenMPT based module player libary and command-line player"
+DESCRIPTION="libopenmpt is a cross platform C++ and C libary to decode tracked \
+music files (modules) int a raw PCM audio stream. openmpt123 is a \
+cross-platform command line or terminal based module file player."
+HOMEPAGE="https://lib.openmpt.org/libopenmpt/"
+COPYRIGHT="2004-2016 OpenMPT contributors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-${portVersion/\~/-}-autotools.tar.gz"
+CHECKSUM_SHA256="a6a7e6da1ae66e1cf46985ee92c182e50652d71b96135e9fa6048e132d844753"
+SOURCE_DIR="libopenmpt-${portVersion%~*}-autotools"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	libopenmpt$secondaryArchSuffix = $portVersion
+	lib:libopenmpt$secondaryArchSuffix = 0.0.20 compat >= 0
+	cmd:openmpt123$secondaryArchSuffix = $portVersion
```

Comment:
> Marked x86_gcc2 arch as broken since the package requires at least GCC 4.1.

OK. This means we can use the commandBinDir "trick" (like you did for app-text/unrtf/unrtf-0.21.9.recipe):
You need to drop the $secondaryArchSuffix after cmd:openmpt123, insert the conditional definition for commandBinDir and change the runConfigure command (as you did for unrtf.)